RxStarscreamTests/RxStarscreamTests.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  RxStarscreamTests.swift
+//  RxStarscreamTests
+//
+//  Created by Cezary Kopacz on 25/03/2017.
+//  Copyright © 2017 Guy Kahlon. All rights reserved.
+//
+
+import XCTest
+import RxSwift
+import RxTest
+import RxStarscream
+import Starscream
+
+extension WebSocketEvent: Equatable { }
+
+public func ==(lhs: WebSocketEvent, rhs: WebSocketEvent) -> Bool {
+    switch (lhs, rhs) {
+    case (.connected, .connected):
+        return true
+    case (.disconnected(let lhsError), .disconnected(let rhsError)):
+        return lhsError == rhsError
+    case (.message(let lhsMsg), .message(let rhsMsg)):
+        return lhsMsg == rhsMsg
+    case (.data(let lhsData), .data(let rhsData)):
+        return lhsData == rhsData
+    case (.pong, .pong):
+        return true
+    default:
+        return false
+    }
+}
+
+class RxStarscreamTests: XCTestCase {
+
+    private var connectedObserver: TestableObserver<Bool>!
+    private var pongObserver: TestableObserver<WebSocketEvent>!
+    private var responseObserver: TestableObserver<WebSocketEvent>!
+    private var socket: WebSocket!
+
+    let disposeBag = DisposeBag()
+
+    override func setUp() {
+        super.setUp()
+
+        socket = WebSocket(url: URL(string: "wss://echo.websocket.org")!)
```

Comment:
Oh gotya :) Cool. That's a nice idea. 
I'm thinking whether or not the force unwrapping is good here but I guess for a test case that should be fine.